bin/node/runtime/src/lib.rs

```diff
@@ -821,11 +821,26 @@ parameter_types! {
 	pub Schedule: pallet_contracts::Schedule<Runtime> = Default::default();
 }
 
+/// The safest default is to allow no calls at all.
+///
+/// Runtimes should whitelist dispatchables that are allowed to be called from contracts
+/// and make sure they are stable. Dispatchables exposed to contracts are not allowed to
+/// change because that would break already deployed contracts. The `Call` structure itself
+/// is not allowed to change the indices of existing pallets, too.
+pub struct ContractsCallFilter;
+impl Filter<Call> for ContractsCallFilter {
+	fn filter(_: &Call) -> bool {
+		false
+	}
+}
+
```

Comment:
I recommend you remove the `()` impl of `Filter` and instead create two default implementations in `frame_support`:

`AllowAllFilter` which returns true, and `AllowNoneFilter` which returns false.

Then we force a user to pick one of these, or implement their own.

Reply:
In any case, if you rather provide this struct than the suggestion above, should be named something like `AllowNoneFilter`

Reply:
I think this is a good idea. I went with `AllowAllFilter` and `DenyAllFilter` because I think those are easier to tell apart visually.

Reply:
Can I make a naming suggestion to just call them `AllowAll` and `DenyAll`?
We could even potentially implement other `Filter`-like things on them (looking at XCM related stuff here)

Reply:
Makes sense. They are already in the `filter` module so the suffix does not make sense anyways (same is true for `FilterStack`I guess).

Reply:
I did the rename. Consider approving the companion, too.